### PR TITLE
DOC: merge up whats_new

### PR DESCRIPTION
**Pull request by tacaswell**

### CODE REVIEW

**Comment by dopplershift** [comment]
Docs are actually failing to build on what's new here. My favorite sphinx-gallery error: "image file not readable"

File: doc/users/whats_new.rst [hunk 1]
```diff
@@ -24,6 +24,65 @@ revision, see the :ref:`github-stats`.
...
+The :meth:`~matplotlib.axes.projections.polar.PolarAxes.set_rorigin` method may
+be used to provide an offset to the minimum plotting radius, producing an
+annulus.
+
+The :meth:`~matplotlib.projections.polar.PolarAxes.set_theta_zero_location` now
+has an optional :code:`offset` argument. This argument may be used to further
+specify the zero location based on the given anchor point.
+
+.. figure:: ../../gallery/pie_and_polar_charts/images/sphx_glr_polar_scatter_001.png
+   :target: ../../gallery/pie_and_polar_charts/polar_scatter.html
```
Comment (dopplershift) [inline doc/users/whats_new.rst]: Does this need to lose one set of ".."?

Comment (tacaswell) [reply doc/users/whats_new.rst]: Yes, waiting for link checking to finish so I can test locally and then push.

**Comment by tacaswell** [comment]
Self-merging on @dopplershift 's previous approval and passing circle.

**Comment by anntzer** [comment]
@tacaswell this https://github.com/hawkowl/towncrier looks like it could be useful to automate this kind of work (well, dunno how overengineered it is compared to doing it manually).

**Pull request merged**